src/music_gui.cpp

```diff
@@ -628,8 +685,8 @@ struct MusicWindow : public Window {
 
 			case WID_M_TRACK_NAME: {
 				Dimension d = GetStringBoundingBox(STR_MUSIC_TITLE_NONE);
-				for (uint i = 0; i < NUM_SONGS_AVAILABLE; i++) {
-					SetDParamStr(0, GetSongName(i));
+				for (auto &song : _music.music_set) {
```

Comment:
i think the generally accepted way of doing these for loops is `const auto &foo` for that const-correctness goodness